cpk-core/ivy.xml

```diff
@@ -48,9 +48,14 @@
     <dependency org="com.sun.jersey" name="jersey-client" rev="1.16" conf="test->default" transitive="false"/>
 
 
+
     <!--  third-party dependencies for tests -->
     <dependency org="com.google.guava" name="guava" rev="17.0" conf="test->default"/>
-     <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho-kettle" name="kettle-json-plugin" rev="${dependency.kettle.revision}" transitive="false" conf="test->default"  changing="true">
+      <artifact name="kettle-json-plugin" type="jar" ext="jar" conf=""/>
```

Comment:
Just a curiosity... I have no idea why but the kettle-json-plugin jar artifact is not showing when searching for it in nexus: http://nexus.pentaho.org/#nexus-search;gav~pentaho-kettle~kettle-json-plugin~~~~kw,versionexpand
However it does resolve properly and it appears in the maven metadata http://nexus.pentaho.org/content/groups/omni/pentaho-kettle/kettle-json-plugin/7.1-SNAPSHOT/maven-metadata.xml. Don't know if its worth it to raise the question to the build team.